Library/Homebrew/os/linux/glibc.rb

```diff
@@ -11,6 +11,14 @@ def system_version
 
         @system_version = Version.new version
       end
+
+      def minimum_version
+        "2.12"
+      end
+
+      def below_minimum_version?
+        system_version < minimum_version
```

Comment:
No, this should still warn I think. If 2.12 is required to pour bottles I think this is fine since users are basically out in the wilderness (building from source). I once ran Linuxbrew on an older version of CentOS and it was not pleasant.